hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TxnReceiptRegression.java

```diff
@@ -113,13 +115,14 @@ private HapiSpec receiptUnavailableIfRejectedInPrecheck() {
                 .then(getReceipt("failingTxn").hasAnswerOnlyPrecheck(RECEIPT_NOT_FOUND));
     }
 
+    @HapiTest
     private HapiSpec receiptNotFoundOnUnknownTransactionID() {
         return defaultHapiSpec("receiptNotFoundOnUnknownTransactionID")
                 .given()
                 .when()
                 .then(withOpContext((spec, ctxLog) -> {
-                    HapiGetReceipt op =
-                            getReceipt(spec.txns().defaultTransactionID()).hasAnswerOnlyPrecheck(RECEIPT_NOT_FOUND);
+                    final HapiGetReceipt op = getReceipt(spec.txns().defaultTransactionID())
+                            .hasAnswerOnlyPrecheck(INVALID_TRANSACTION_ID);
```

Comment:
We should not be changing the response and comply with mono-service

Reply:
That is correct, the test logic should not change. We need to align the behavior in the modular code to match the mono service behavior, and the tests are the guard to ensure we do it correctly.

Reply:
Understood. Changes are applied.